src/test/java/ipman/commands/DeleteCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package ipman.commands;
+
+import ipman.models.DummyFileManager;
+import ipman.models.TaskList;
+import ipman.models.ToDo;
+import ipman.ui.SpyUi;
+import org.junit.jupiter.api.Test;
```

Comment:
Perhaps similar import statements should be grouped together?

Reply:
You're right, I realised my checkstyle did not run on test classes. I have corrected this.